src/main/java/Bart.java

```diff
@@ -0,0 +1,86 @@
+import java.util.Scanner;
+
+public class Bart {
+    private static final String LINE = "____________________________________________________________";
+    private static final Task[] tasksArray = new Task[100];
+    private static int taskCount = 0;
+
+    public static void main(String[] args) {
+        greetUser();
+        manageTask();
+        byeUser();
+    }
+    public static void greetUser() {
+        System.out.println(LINE);
+        System.out.println("Hello! I'm Bartholomew, but you can call me Bart for short :)");
+        System.out.println("What can I do for you?");
+        System.out.println(LINE);
+    }
+    public static void manageTask() {
+        Scanner in = new Scanner(System.in);
+        String command = "";
+        while (!command.equals("bye")) {
+            command = in.nextLine();
+            if (command.equals("list")) {
+                listTasks();
+            } else if (command.startsWith("mark")) {
+                markTask(command);
+            } else if (command.startsWith("unmark")) {
+                unmarkTask(command);
+            } else {
+                if (taskCount < tasksArray.length) {
+                    tasksArray[taskCount] = new Task(command); // Create new Task object
+                    taskCount++;
+                    System.out.println(LINE);
+                    System.out.println("added: " + command);
+                    System.out.println(LINE);
+                } else {
+                    System.out.println("Sorry, task list is full.");
+                }
+            }
+        }
+    }
+    public static void listTasks() {
+        System.out.println(LINE);
+        //Edge case: If list empty
```

Comment:
Nice work in setting aside the edge case.

Just a (really) minor aesthetic edit 
```suggestion
        // Edge case: If list empty
```